src/createAppContainer.js

```diff
@@ -37,6 +45,17 @@ function validateProps(props) {
   }
 }
 
+async function defaultNavigationStatePersister(navState, persistenceKey) {
+  if (!persistenceKey) {
+    return;
+  }
+  await AsyncStorage.setItem(persistenceKey, JSON.stringify(navState));
```

Comment:
We should probably also move the require statement inside here to avoid the warning from RN. cc @brentvatne

Reply:
I added these functions so that the code is backward compatible, but I could as well remove the default value for `persistNavigationState`, `loadNavigationState` and the `persistenceKey` prop, to remove the AsyncStorage dependency altogether. That will be breaking, but in the docs, the api is described as experimental and it is explicitly said the api may change in future, so it's fine, imho.

Reply:
yeah i think it's fine to remove it, and in the docs we can provide some default implementation